catalog/app/containers/Bucket/requests.js

```diff
@@ -59,6 +107,8 @@ export const bucketListing = ({ s3, bucket, path = '', prefix }) =>
 
 const MAX_BANDS = 10
 
+/* eslint-disable no-console */
```

Comment:
i don't think this is a good idea, bc console calls that were accidentally left will be ignored

Reply:
Please take a look https://github.com/quiltdata/quilt/pull/1882

Reply:
i did, and we'll get to it soon-ish, but it seems outside of this pr's context